modules/tinymce/src/themes/silver/main/ts/ui/sidebar/Sidebar.ts

```diff
@@ -101,10 +101,13 @@ const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig, showSi
     Replacing.set(slider, [ makeSidebar(panelConfigs) ]);
 
     // Show the default sidebar
-    if (Type.isString(showSidebar)) {
+    const configKey = Optional.from(showSidebar).map((value) => value.toLowerCase()).getOrUndefined();
```

Comment:
```suggestion
    const configKey = value?.toLowerCase();
```
This can be simplified to this I believe?